GitSpace/Sources/Models/Report.swift

```diff
@@ -24,11 +24,19 @@ struct Report: Codable {
         case knock = "Knock"
     }
     
-    enum ReportReason: String, Codable {
+    enum ReportReason: String, Codable, CaseIterable {
         case spamming = "Spamming"
         case offensive = "Verbal Abuse, Offensive Language"
         case sexual = "Sexual Description(Activity)"
         case cheating = "Cheating"
         case bullying = "Cyberbullying or Harassment"
```

Comment:
```suggestion
        case bullying = "Cyberbullying or Harassment"
        
        func makeDescription() -> String {
            switch self {
            case .spamming:
                return "~~~ Description"
            case .offensive:
                return "~~~ Description"
            case .sexual:
                return "~~~ Description"
            case .cheating:
                return "~~~ Description"
            case .bullying:
                return "~~~ Description"
            }
        }
```
메서드를 사용한다면 description을 위한 새로운 열거형을 만들지 않고 이런 느낌으로 작성할 수 있을 것 같습니다 😁

Reply:
저도 제균님이 제안한 방법대로 메서드를 사용하면 좋을 것 같아요!
두 열거형이 서로 연관되어 있으니 하나의 열거형 안에서 메서드로 관리하는 게 유지보수 할 때에도 좋아보입니다!

Reply:
저도 제균님이 제안한 방법대로 메서드를 사용하면 좋을 것 같아요!
두 열거형이 서로 연관되어 있으니 하나의 열거형 안에서 메서드로 관리하는 게 유지보수 할 때에도 좋아보입니다!